R/copernicus_article.R

```diff
@@ -0,0 +1,129 @@
+                         md_extensions = c(
+                           "-autolink_bare_uris", # disables automatic links, needed for plain email in \correspondence
+                           "-auto_identifiers"    # disables \hypertarget commands
+                           )) {
+  if (inherits(base_format, "character")) {
+    FMT <- eval(parse(text = base_format))
+  } else {
+    FMT <- match.fun(base_format)
+  }
+  FMT(...,
+      citation_package = citation_package,
+      keep_tex = keep_tex,
+      md_extensions = md_extensions,
+      template = find_resource("copernicus_article", "template.tex"))
+}
+
+# quick dev shortcut for Ubuntu: click "Install and restart" then run:
+# unlink("MyArticle/", recursive = TRUE); draft("MyArticle.Rmd", template = "copernicus_article", package = "rticles", edit = FALSE); render("MyArticle/MyArticle.Rmd"); system(paste0("gnome-open ", here::here("MyArticle", "MyArticle.pdf")))
+
+copernicus_journals <- list(
+  "Advances in Geosciences" = "adgeo",
+  "Advances in Radio Science" = "ars",
+  "Advances in Science and Research" = "asr",
+  "Advances in Statistical Climatology, Meteorology and Oceanography" = "ascmo",
+  "Annales Geophysicae" = "angeo",
+  "Archives Animal Breeding" = "aab",
+  "ASTRA Proceedings" = "ap",
+  "Atmospheric Chemistry and Physics" = "acp",
+  "Atmospheric Measurement Techniques" = "amt",
+  "Biogeosciences" = "bg",
+  "Climate of the Past" = "cp",
+  "Drinking Water Engineering and Science" = "dwes",
+  "Earth Surface Dynamics" = "esurf",
+  "Earth System Dynamics" = "esd",
+  "Earth System Science Data" = "essd",
+  "E&G Quaternary Science Journal" = "egqsj",
+  "Fossil Record" = "fr",
+  "Geographica Helvetica" = "gh",
+  "Geoscience Communication" = "gc",
+  "Geoscientific Instrumentation, Methods and Data Systems" = "gi",
+  "Geoscientific Model Development" = "gmd",
+  "History of Geo- and Space Sciences" = "hgss",
+  "Hydrology and Earth System Sciences" = "hess",
+  "Journal of Micropalaeontology" = "jm",
+  "Journal of Sensors and Sensor Systems" = "jsss",
+  "Mechanical Sciences" = "ms",
+  "Natural Hazards and Earth System Sciences" = "nhess",
+  "Nonlinear Processes in Geophysics" = "npg",
+  "Ocean Science" = "os",
+  "Primate Biology" = "pb",
+  "Proceedings of the International Association of Hydrological Sciences" = "piahs",
+  "Scientific Drilling" = "sd",
+  "SOIL" = "soil",
+  "Solid Earth" = "se",
+  "The Cryosphere" = "tc",
+  "Web Ecology" = "we",
+  "Wind Energy Science" = "wes"
+)
+
+#' Copernicus journal abbreviations
+#'
+#' Get the journal abbreviation for all journals supported by the copernicus article template.
+#'
+#' @param name A regular expression to filter the by the journal name, see \code{pattern} in \code{\link[base]{grep}}; defaults to \code{*}.
+#'
+#' @examples
+#' names(copernicus_journal_abbreviations())
+#' copernicus_journal_abbreviations(name = "Science Data")
+#'
+#' @export
```

Comment:
I'd merge the doc of this function with `copernicus_article`, i.e.

```r
#' @rdname copernicus_article
```

Reply:
Done.